db/migrate/20150827092935_add_opt_in_field_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOptInFieldToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :opt_in_for_research, :boolean, default: true
```

Comment:
is the default suppose to be true? 
and if the user unticks does it save the value as false?